app/views/streams/main_stream.html.haml

```diff
@@ -32,17 +32,28 @@
 
       %ul#stream_selection
         %li{data: {stream: "stream"}}
-          = link_to t("streams.multi.title"), stream_path, rel: "backbone", class: "hoverable"
+          = link_to t("streams.multi.title"), stream_path, rel: "backbone",
```

Comment:
These don't mean to not use `"foo-bar" => baz`, These mean to not mix `foo: bar` and `foo => bar`